src/main/webapp/script.js

```diff
@@ -41,19 +41,57 @@ function submitAlert() {
 async function getGuide() {
     const responseFromServer = await fetch('/guides');
     const jsonResponse = await responseFromServer.json();
-    const guides_container = document.getElementById($('#product'));
 
     for (let i = 0; i < jsonResponse.length; i++) {
         $('#product').append(guidesToHTML(jsonResponse[i]));
     }
 }
 
 function guidesToHTML(guide) {
+    console.log("guide.id:" + guide.id);
     return `<div class="card w-75 itemBox ${guide.category}">
+        <div class="card-header">${guide.id}</div>
+        <a href="guide.html?id=${guide.id}">
+            <div class="card-body">
+                <h5 class="card-title">${guide.title}</h5>
+                <p class="card-text">${guide.description}</p>
+                <footer class="blockquote-footer">
+                    <small class="text-muted">
+                    Last updated <cite title="Source Title">${new Date(guide.timestamp).toDateString()}</cite>
+                    </small>
+                </footer>
+            </div>
+        </a>
+    </div>`;
+}
+
+function getGuideId() {
+    let urlString  = window.location.href;
+    var url = new URL(urlString);
+    var id = url.searchParams.get("id");
+    console.log(id);
+    return id;
+}
+
+function onLoadGuide() {
+    addGuidePage(getGuideId());
+}
+
+async function addGuidePage(id) {
+    const guideName = "/guides/" + id;
+    console.log("guide Name = " + guideName);
```

Comment:
Nit: I'm not sure this is necessary, since it will show up in the network requests.

Reply:
Ok thanks, I'll fix that!